docs/operator-manual/reconcile.md

```diff
@@ -111,3 +111,56 @@ data:
     # actually changing in content.
     - .status.conditions[].lastTransitionTime
 ```
+
+## Completely ignore resources update
```

Comment:
You should update current sections of the docs in this page that mention that ignore resource updates is not supported for resources that are not tracked.